### PR TITLE
Updates to polling

### PR DESCRIPTION
**Pull request by keithing**
The `_poller` and `_polling_args` attributes of `PollableResult` might be generally useful when building custom workflows. This PR exposes these attributes as public to designate them as safe to access.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - civis.io functions use the "hidden" API option to keep jobs out of the UI. Deprecate the "archive" parameter in favor of "hidden".
 - civis.io.query_civis now has a "hidden" parameter which defaults to True
+- expose `polling` and `polling_args` as public attribute in PollableResults
```
Comment (stephen-hoover) [inline CHANGELOG.md]: "`poller` and `poller_args`"?
"attribute" --> "attributes"
"PollableResults" --> "PollableResult"
Backticks around "PollableResult"?

File: civis/polling.py [hunk 2]
```diff
@@ -38,6 +38,19 @@ class PollableResult(futures.Future):
     polling_interval : int or float
         The number of seconds between API requests to check whether a result
         is ready.
+
+    Examples
+    --------
+    >>> client = civis.APIClient()
+    >>> database_id, sql, preview_rows, cred_id = 999, "SELECT 1", 10, 999
```
Comment (stephen-hoover) [inline civis/polling.py]: Would it be useful to use `client.get_database_id` and `client.get_database_credential_id` in this example, or is that out of scope?

File: CHANGELOG.md (hunk 1 above)
Comment (keithing) [inline CHANGELOG.md]: Good catch.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - civis.io functions use the "hidden" API option to keep jobs out of the UI. Deprecate the "archive" parameter in favor of "hidden".
 - civis.io.query_civis now has a "hidden" parameter which defaults to True
+- expose `polling` and `polling_args` as public attributes in `PollableResults`
```
Comment (stephen-hoover) [inline CHANGELOG.md]: The attribute names should be `poller` and `poller_args`.

**Comment by stephen-hoover** [comment]
LGTM!

**Pull request merged**